core/src/replay_stage.rs

```diff
@@ -889,6 +889,11 @@ impl ReplayStage {
                 if child_stats.is_locked_out || !child_stats.vote_threshold {
                     continue;
                 }
+                datapoint_info!(
+                    "replay_stage-pick_best_fork",
+                    ("slot", best_bank.slot(), i64),
+                    ("weight", child_stats.weight, i64),
```

Comment:
i64 is probably going to be too small, the weight is a u128. In my PR I was converting this to hex, don't know if there's a better solution since influx doesn't support a bigger datatype.

Reply:
f64 maybe